src/features/subdomains/SubdomainTableCard.tsx

```diff
@@ -0,0 +1,61 @@
+//- React Imports
+import { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+
+//- Library Imports
+import { ethers } from 'ethers';
+import { Domain, TokenPriceInfo } from '@zero-tech/zns-sdk';
+
+//- Hook Imports
+import { useDomainMetadata } from '../../lib/hooks/useDomainMetadata';
+import { useDomainMetrics } from '../../lib/hooks/useDomainMetrics';
+import { formatEthers, formatNumber } from '../../lib/util/number/number';
+import { useBuyNowPrice } from '../../lib/hooks/useBuyNowPrice';
+
+type SubdomainTableCardProps = {
+	domain: Domain;
+	paymentTokenData: TokenPriceInfo;
+};
+
+const SubdomainTableCard: FC<SubdomainTableCardProps> = ({
+	domain,
+	paymentTokenData,
+}) => {
+	const { push: goTo } = useHistory();
```

Comment:
Any reason  to replace the push function? I would to use `const history = useHistory(); history.push('/my_page_path')`

Reply:
This was probably just legacy from code that I wrote, haha. @domw30 I agree with Joon here

Reply:
ha yeah pulled from the existing repo. Shall update